docs/install/osx_setup.md

```diff
@@ -146,10 +157,11 @@ We have installed MXNet core library. Next, we will install MXNet interface pack
 To install the MXNet Python binding navigate to the root of the MXNet folder then run the following:
 
 ```bash
-$ cd python
-$ pip install -e .
+virtualenv -p`which python3` py3_venv
+source py3_venv/bin/activate
+pip install -e python
 ```
-
+First we create a [virtualenv](https://docs.python-guide.org/dev/virtualenvs/#lower-level-virtualenv) to isolate this installation from our global environment.
```

Comment:
Same for virtualenv. This has nothing to do with MXNet. We should let users to choose their own tool (such as conda) instead.